### DISCUSSION TITLE
How to remove pruned branches?

### DISCUSSION THREAD

**Discussion by ghost** [jesseduffield/lazygit#1647] [Q&A] [answered]
Hi,

Is there a way to remove pruned branches from the list of `Local Branches` in lazygit?
When I do `git fetch -p`, old branches are still visible. I would like to somehow remove them.

**Comment by screendriver** [comment]
I do that with a custom command [ctrl+d](https://github.com/jesseduffield/lazygit/wiki/Custom-Commands-Compendium#pruning-deleted-remote-branches)

**Comment by ghost** [comment, marked as answer]
This command will clean up local branches:

```
git fetch -p && for branch in $(git branch -vv | grep ': gone]' | awk '{print $1}'); do git branch -D $branch; done
```
